nextflow.config

```diff
@@ -1041,6 +1043,17 @@ profiles {
     test { includeConfig 'conf/test.config' }
 }
 
+// Set default registry for Apptainer, Docker, Podman and Singularity independent of -profile
+// Will not be used unless Apptainer / Docker / Podman / Singularity are enabled
+// Set to your registry if you have a mirror of containers
+apptainer.registry   = 'quay.io'
+docker.registry      = 'quay.io'
+podman.registry      = 'quay.io'
+singularity.registry = 'quay.io'
+
+// Override the default Docker registry when required
+process.ext.override_configured_container_registry = false
```

Comment:
By default this should be `true` instead of `false`.

Reply:
Yes! Must have slipped back in from testing. Good catch!